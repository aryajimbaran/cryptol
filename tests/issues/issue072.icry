:sat \x -> x > 0x4
:t it
it
:set satNum = all
:sat \x -> x > 0x4
:t it
it
:set satNum = foobar
:set satNum = 0
:set satNum = 3
:sat \x -> x > 0x4
:t it
it