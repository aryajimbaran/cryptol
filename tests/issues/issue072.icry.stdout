Loading module Cryptol
(\x -> x > 0x4) 0x5 = True
it : {result : Bit, arg1 : [4]}
{result = True, arg1 = 0x5}
(\x -> x > 0x4) 0x5 = True
(\x -> x > 0x4) 0x6 = True
(\x -> x > 0x4) 0x7 = True
(\x -> x > 0x4) 0x8 = True
(\x -> x > 0x4) 0x9 = True
(\x -> x > 0x4) 0xa = True
(\x -> x > 0x4) 0xb = True
(\x -> x > 0x4) 0xc = True
(\x -> x > 0x4) 0xd = True
(\x -> x > 0x4) 0xe = True
(\x -> x > 0x4) 0xf = True
it : [11]{result : Bit, arg1 : [4]}
[{result = True, arg1 = 0x5}, {result = True, arg1 = 0x6},
 {result = True, arg1 = 0x7}, {result = True, arg1 = 0x8},
 {result = True, arg1 = 0x9}, {result = True, arg1 = 0xa},
 {result = True, arg1 = 0xb}, {result = True, arg1 = 0xc},
 {result = True, arg1 = 0xd}, {result = True, arg1 = 0xe},
 {result = True, arg1 = 0xf}]
must be an integer > 0 or "all"
must be an integer > 0 or "all"
(\x -> x > 0x4) 0x5 = True
(\x -> x > 0x4) 0x6 = True
(\x -> x > 0x4) 0x7 = True
it : [3]{result : Bit, arg1 : [4]}
[{result = True, arg1 = 0x5}, {result = True, arg1 = 0x6},
 {result = True, arg1 = 0x7}]
